components/sidebar/sidebar_channel/sidebar_channel.tsx

```diff
@@ -143,6 +143,7 @@ export default class SidebarChannel extends React.PureComponent<Props, State> {
             multiSelectedChannelIds,
             autoSortedCategoryIds,
         } = this.props;
+        console.log('render sidebar channel ' + channel.id);
```

Comment:
seems like a debug statement, right? if so we should remove it

Reply:
I removed it in a later commit

Reply:
my bad, I was going through commits separately to keep apart the changes